### ISSUE TITLE
Convert entries with non-unique data into psuedo-unique data

### ISSUE THREAD

**Issue by theanine** [theanine/cooccurence#1]
E.g.

This original data:
```
	data = [[1, 0, 3, 0],
	        [0, 1, 2, 0],
	        [0, 1, 1, 0]]
```

Should become this:
```
	data = [[1, 0, 1, 1, 1, 0],
	        [0, 1, 1, 1, 0, 0],
	        [0, 1, 1, 0, 0, 0]]
```

Mappings should remain consistent.

**Comment by theanine** [comment]
We need this because the underlying cosine similarity calculation doesn't seem to work with non-unique data.

**Comment by theanine** [comment]
Fixed in #7.
